V2/gameFuseLeaderboardEntry.js

```diff
@@ -41,5 +41,44 @@ class GameFuseLeaderboardEntry {
             }, {});
         return dictionary;
     }
+
+    // method to get a user's full profile data
```

Comment:
Should we do this right now or can we wait to see if we actually need it?

Reply:
i think we talked about this but no we will wait for getting the full user data method